sdk/core/CHANGELOG.md

```diff
@@ -1,3 +1,77 @@
+# release 6.0.0-beta.6 (2021-08-13)
+* Simplified npm dependencies; it is enough to only install `@affinidi/wallet-{PLATFORM}-sdk` now (without other `@affinidi` packages);
+  all the relevant types and methods are now exported from `@affinidi/wallet-{PLATFORM}-sdk`.
+* Introduced new streamlined set of factories and methods for user management, with clearer return values.
+  - Note that instances returned by non-legacy factories do not have legacy methods, and vice versa.
+  - Note that in new methods, every method accepting confirmation code now also requires to be passed a token,
+    which is the string value returned by a method that sent a confirmation code.
+  - Note that new instance (non-static) methods no longer accept an options object.
+  - Note that signup methods that return "either `string` or `AffinidiWallet`" are deprecated,
+    instead you should either let user to explicitly choose between using an username, email address or phone number,
+    or detect the type of login by invoking `Util.getLoginType`,
+    and then invoke the relevant signup method (`signUpWithUsername` or `initiateSignUpByEmail` or `initiateSignUpByPhone`).
+* Implemented new methods:
```

Comment:
@inga-affinidi I believe it's not a full list of new methods, correct? So I am as a consumer, I can get that all my methods deprecated, but what I should use instead? I believe at least we should add a reference to the README (e.g. - new interface check here)

Reply:
@DenisPopov15 , deprecated methods are marked as deprecated with `@deprecated` comments pointing to the specific non-deprecated alternatives.